src/NHibernate/Event/Default/AbstractFlushingEventListener.cs

```diff
@@ -196,7 +197,7 @@ protected virtual void PrepareEntityFlushes(IEventSource session)
 				Status status = entry.Status;
 				if (status == Status.Loaded || status == Status.Saving || status == Status.ReadOnly)
 				{
-					CascadeOnFlush(session, entry.Persister, me.Key, Anything);
+					CascadeOnFlush(session, entry.Persister, me.Key, anything);
```

Comment:
I do not get it. The property implementation just always directly yield null, and is not overridden anywhere to do anything else. What does this change?

Reply:
> not overridden anywhere

Not until #2752. Looks like you check on 5.3 codebase.

Reply:
Yes, as Roman said: it was overridden in #2752:

https://github.com/nhibernate/nhibernate-core/blob/170210aa211a522f145c61437d4429af794f1f7f/src/NHibernate/Event/Default/DefaultDirtyCheckEventListener.cs#L16

See the discussion starting in https://github.com/nhibernate/nhibernate-core/issues/3307#issuecomment-1562296466 for more context.

Reply:
Indeed, I was not on the adequate code base.